k.makoyid/oop_code/Data_loader

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+import tensorflow as tf
+from tensorflow.keras.datasets import cifar10
+
+
+class DataLoader:
+    @staticmethod
+    def load_cifar10_data():
```

Comment:
What is the purpose of this class? Current implementation contains only one function which can be used without this class and it will be fine

Reply:
Also, fix the file format